cdk/lib/stack.py

```diff
@@ -22,7 +22,18 @@ def __init__(
 
         super().__init__(scope, id, **kwargs)
 
-        vpc = ec2.Vpc(self, "CtfVPC", max_azs=2)
+        preferred_az = f"{aws.AWS_REGION}a"
+
+        vpc = ec2.Vpc(
+            self,
+            "CtfVPC",
+            max_azs=2,
```

Comment:
2 availability zones are required if you are using an application load balancer, which we are